docs/nv2-bicep.md

```diff
@@ -0,0 +1,72 @@
+# Notary v2 - Remote Signing and Verification with Gatekeeper, Ratify and AKS - Using Bicep
+
+
+## Install the notation cli and azure-kv plugin
+
+Starting from the root of this repo, make sure you are logged into the Azure CLI. This is a pre-requisite.
+
+  ```bash
+  az login
+  ```
+
+Set the desired subscription.
+
+  ```bash
+  az account set --subscription <id or name>
+  ```
+
+Once you have done the above, you are ready to deploy the required infrastructure by running the provided `setup.sh` script. The script will prompt you for the following:
+
+
+1. Key Name: **Required** Key name used to sign and verify.
+2. Key Subject Name: **Required** Key subject name used to sign and verify.
+
+
+The script also supports overriding the following arguments, in the following order:
+1. Resource Group Name: This will be the resource group created in Azure. If you do not provide a value `myakv-akv-rg` will be used.
+2. Kubernetes Version: This is the version of Kubernetes control plane. If you do not provide a value, the latest Kubernetes version available for the provided location will be used.
+
+The default location for this script is `southcentralus`. As of March 2022, this is the only region that supports [ORAS Artifact][oras-artifact] support, which enables storing signatures and other types of references.  
+See [ACR Support for ORAS Artifacts][acr-oras-support] for more details.
+
+Bash without overrides
+
+  ```bash
+  ./setup.sh
+  ```
+Bash with overrides
+
+  ```bash
+  ./setup.sh my-akvrg-override 1.23.3
+  ```
+
```

Comment:
Now that the script executes [all of the flow](https://github.com/Azure/notation-azure-kv/pull/17/files#diff-4209d788ad32c40cbda3c66b3de47eefb929308ca703bb77a6382625986add17R345-R353) you've got nothing really to *show* or explain.  

I suggest you break out the buildImageandSign functions so you can explain/show them (versus auto execution) or at a bare minimum put some text in here to have the user see the signed image.